todo_app/app.py

```diff
@@ -33,6 +44,7 @@ def index():
 
     def user_is_writer(current_user):
         if app.config['LOGIN_DISABLED']:
+            app.logger.info("Login")
```

Comment:
This log message seems meaningless or even a lie.

Should be something like "Login is disabled"